catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -0,0 +1,212 @@
+                }
+
+                String[] pathParts = filename.split("/");
+                String id = pathParts[ID];
+                String type = pathParts[TYPE];
+
+                switch (type) {
+                case "metacard": {
+                    String metacardName = pathParts[NAME];
+                    Metacard metacard = null;
+                    try {
+                        metacard = transformer.transform(new UncloseableBufferedInputStreamWrapper(
+                                zipInputStream), id);
+                    } catch (IOException | CatalogTransformerException e) {
+                        LOGGER.debug("Could not transform metacard: {}", id);
+                    }
+                    catalogProvider.create(new CreateRequestImpl(metacard));
+                    break;
+                }
+                case "content": {
+                    String contentFilename = pathParts[NAME];
+                    ContentItem contentItem = new ContentItemImpl(id,
+                            new ZipEntryByteSource(new UncloseableBufferedInputStreamWrapper(zipInputStream)),
+                            null,
+                            contentFilename,
+                            entry.getSize(),
+                            null);
+                    CreateStorageRequestImpl createStorageRequest = new CreateStorageRequestImpl(
+                            Collections.singletonList(contentItem),
+                            id,
+                            new HashMap<>());
+                    storageProvider.create(createStorageRequest);
+                    storageProvider.commit(createStorageRequest);
+                    break;
+                }
+                case "derived": {
+                    String qualifier = pathParts[NAME];
+                    String derivedContentName = pathParts[DERIVED_NAME];
+                    ContentItem contentItem = new ContentItemImpl(id,
+                            qualifier,
+                            new ZipEntryByteSource(new UncloseableBufferedInputStreamWrapper(zipInputStream)),
+                            null,
+                            derivedContentName,
+                            entry.getSize(),
+                            null);
+                    CreateStorageRequestImpl createStorageRequest = new CreateStorageRequestImpl(
+                            Collections.singletonList(contentItem),
+                            id,
+                            new HashMap<>());
+                    storageProvider.create(createStorageRequest);
+                    storageProvider.commit(createStorageRequest);
+                    break;
+                }
+                default: {
+                    LOGGER.debug("Cannot interpret type of {}", type);
+                }
+                }
+
+                entry = zipInputStream.getNextEntry();
+            }
+        }
+        console.println("File imported successfully.");
```

Comment:
The export command logs the number of content items and metacards exported.  Ideally the import command would log the same.